routers/api/v1/api.go

```diff
@@ -617,7 +617,7 @@ func Routes(ctx gocontext.Context) *web.Route {
 			// setting.CORSConfig.AllowSubdomain // FIXME: the cors middleware needs allowSubdomain option
 			AllowedMethods:   setting.CORSConfig.Methods,
 			AllowCredentials: setting.CORSConfig.AllowCredentials,
-			AllowedHeaders:   []string{"Authorization", "X-Gitea-OTP"},
+			AllowedHeaders:   setting.CORSConfig.Headers,
```

Comment:
```suggestion
			AllowedHeaders:   append(setting.CORSConfig.Headers, "X-Gitea-OTP"),
```
Should `Authorization` be forced too?

Reply:
I do think so, yeah